eng/pipelines/mgmt.yml

```diff
@@ -15,6 +15,7 @@ pr:
     - sdk/attestation/Microsoft.Azure.Management.Attestation
     - sdk/authorization/Microsoft.Azure.Management.Authorization
     - sdk/automation/Microsoft.Azure.Management.Automation
+     - sdk/automanage/Microsoft.Azure.Management.AutoManage
```

Comment:
```suggestion
    - sdk/automanage/Microsoft.Azure.Management.AutoManage
```

Reply:
@allenjzhang without this spacing fix I suspect this pipeline will be broken. yml is very specific about the indention.